api/v1beta1/verticadb_types.go

```diff
@@ -356,6 +356,13 @@ type VerticaDBSpec struct {
 	// here are applied to the default startup probe we create. If this is
 	// omitted, we use the default probe.
 	StartupProbeOverride *corev1.Probe `json:"startupProbeOverride,omitempty"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:ServiceAccount","urn:alm:descriptor:com.tectonic.ui:advanced"}
+	// +kubebuilder:validation:Optional
+	// The name of a serviceAccount to use to run the Vertica pods. If this is
+	// omitted or the serviceAccount doesn't exist, the operator will create one
+	// along with a Role and RoleBinding.
+	ServiceAccountName string `json:"serviceAccountName,omitempty"`
```

Comment:
Is it worth mentioning that "If this is omitted or the serviceAccount does't exist, the operator will create one with respectively an internal generated name or the  specified name, as well as a Role and RoleBinding."

Reply:
How about this?
If the service account is not specified or does not exist, the operator will create one, using the specified name if provided, along with a Role and RoleBinding.